data/human/outfits.txt

```diff
@@ -391,6 +391,16 @@ outfit "Nerve Gas"
 	"unplunderable" 1
 	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."
 
+outfit "Drugs"
+	plural "Drugs"
+	category "Hand to Hand"
+	cost 10000
+	thumbnail "outfit/Drugs"
```

Comment:
```suggestion
	thumbnail "outfit/drugs"
```


Afaik file name should be all lowercase. So apply this change to the file name as well.